scripts/2.mutation-classifier.py

```diff
@@ -48,46 +70,76 @@
 
 # In[4]:
 
-get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'expression-matrix.tsv.bz2')\nexpression_df = pd.read_table(path, index_col=0)")
+
+path = os.path.join('download', 'expression-matrix.tsv.bz2')
+expression_df = pd.read_table(path, index_col=0)
+
+path = os.path.join('download', 'mutation-matrix.tsv.bz2')
+mutation_df = pd.read_table(path, index_col=0)
+
+path = os.path.join('download', 'covariates.tsv')
+covariate_df = pd.read_table(path, index_col=0)
 
 
 # In[5]:
 
-get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'mutation-matrix.tsv.bz2')\nmutation_df = pd.read_table(path, index_col=0)")
+
+# Select acronym_x and n_mutations_log1p covariates only
+disease_cols = [col for col in covariate_df.columns if col.startswith('acronym_')]
+
+# Filter covariate columns by disease if a list was provided
+if disease_acronyms:
+    disease_cols = [col for col in disease_cols if col.endswith(tuple(disease_acronyms))]
+    
+selected_cols = disease_cols + ['n_mutations_log1p']
+covariate_df = covariate_df[selected_cols]
 
 
 # In[6]:
 
+
 get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'expression-genes.tsv')\nexpression_genes_df = pd.read_table(path, index_col=0)")
```

Comment:
Let's also get rid of `%%time` here. And consider moving this to the load data cell.